modules/geo/vector.ts

```diff
@@ -1,5 +1,8 @@
+export type Vec2 = [x: number, y: number];
+export type Vec3 = [x: number, y: number, z: number];
```

Comment:
`Vec3` is not used anywhere as far as I can see, is it? I also can't think of a situation where we'd need it[^1] in the near future, so I'd probably omit it for now.

[^1]: except for the zoom/x/y `map` parameter, but that's not really a vector, and the order would be different again.